packages/main/src/SegmentedButton.ts

```diff
@@ -64,13 +66,17 @@ type SegmentedButtonSelectionChangeEventDetail = {
  *
  * @event sap.ui.webc.main.SegmentedButton#selection-change
  * @param {HTMLElement} selectedItem the pressed item.
+ * Note: deprecated since 1.14.0 and will be removed in the next major release, use the <code>pressedItems</code> parameter instead.
+ * @param {string} pressedItems an array of pressed items. Note: available since 1.14.0.
  * @public
  */
 @event("selection-change", {
 	detail: {
 		selectedItem: { type: HTMLElement },
+		pressedItems: { type: Array },
```

Comment:
As the mode values will be `SingleSelect` and `MultiSelect`, the event is called `selection-change`, we suggest renaming everything from `pressedItems` to `selectedItems`

Reply:
done